packages/vertix-base/src/modules/service/service-base.ts

```diff
@@ -0,0 +1,54 @@
+import { ObjectBase } from "@vertix.gg/base/src/bases/object-base";
```

Comment:
/ask create a todo issue to move this module(service) to another repo called @zenflux/core